src/utilities/singleChoice.js

```diff
@@ -0,0 +1,19 @@
+import { singleAnswerQuestion } from './singleAnswer';
+
+export class choiceQuestion extends singleAnswerQuestion {
```

Comment:
The class name should start with an uppercase letter - `ChoiceQuestion`. But since you called this file `singleChoice.js` I think you meant to use the `SingleChoiceQuestion` class name